lib/errorlogger.cpp

```diff
@@ -611,7 +611,7 @@ std::string ErrorLogger::ErrorMessage::FileLocation::stringify() const
 {
     std::ostringstream oss;
     oss << '[' << Path::toNativeSeparators(mFileName);
-    if (line != 0)
+    if (line != Suppressions::Suppression::NO_LINE)
```

Comment:
This change is not related to the cucumber library.

Reply:
correct, only https://github.com/danmar/cppcheck/pull/1383/commits/3ae42658f74fbb00f508a2ad3910cf9ed3ae3011 should be considered for the cucumber thing. did I make the request wrong?

Reply:
yes it seems it got wrong. I would try rebase.

If you type `git rebase -i master`. then you should see something like:

    pick 361b233 commit1
    pick 89349fa commit2
    ...
    pick 3ae4265 last commit

since you just want to keep the last, for the commits to delete change "pick" to "delete":

    delete 361b233 commit1
    delete 89349fa commit2
    ...
    pick 3ae4265 last commit

then save the changes and exit. you should now see a second screen where you choose the commit message. Type something good and the save and exit.

Now type `git push -f` to push the changes..

Reply:
I can't seem to be able to fix this pull request -_-' maybe we should close this one and I'll make a new clean branch for this and issue a new pull request ?